### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by knowbody**
add *get started* instructions

### CODE REVIEW

**Comment by boygirl** [comment]
Approved!

File: README.md [hunk 1]
```diff
@@ -11,6 +11,35 @@ See the docs and examples on the website: http://victory.formidable.com
...
+        <VictoryPie />
+      );
+    }
+  }
+  
+  render(<PieChart />, document.getElementById('app'));
+  ```
+
+3. `VictoryPie` component will be rendered, and you should see:  
+![VictoryPie](http://i.imgur.com/srEoGJv.png)
```
Comment (coopy) [inline README.md]: Prefer hosting images on Github: Drag an image to a comment anywhere (PR, issue), and copy the resulting URL. You don't have to actually post the comment.

**Comment by coopy** [comment]
Please rebase and collapse down to one commit that says "add _get started_ instructions to README"

**Comment by knowbody** [comment]
@coopy good to go now :rocket:

**Comment by coopy** [comment]
Thanks! :+1:

**Pull request merged**